execute/plugin.go

```diff
@@ -178,6 +178,7 @@ func getPendingExecutedReports(
 	return groupedCommits, nil
 }
 
+//nolint:gocyclo
```

Comment:
Can we have a low prio to fix this, we shouldn't have nolint on gocyclo in an ideal scenario